Resources/Prototypes/GameRules/unknown_shuttles.yml

```diff
@@ -1,3 +1,25 @@
+# Add your new event to this RandomRule if you want it to happen via the basic/ramping schedulers.
+- type: entity
+  parent: BaseGameRule
+  id: RandomUnknownShuttleRule
+  components:
+  - type: RandomRule
+    selectableGameRules:
+    # Make the probability sum to 1 or I will remove your fingernails with toothpicks.
```

Comment:
same thing here. there's no point in requiring it to sum to one and doing that will just lead to breaking changes on this file for no reason

Reply:
Remove the 0.25 probs and make the 0.3 prob 1.2.